src/Network.cpp

```diff
@@ -869,17 +869,28 @@ T relative_difference(T a, T b) {
 
 void compare_net_outputs(std::vector<float>& data,
                          std::vector<float>& ref) {
+    // Divide by an additional 2 as we're comparing 2 heads at a time
+    constexpr int64 min_correct_expansions = SELFCHECK_MIN_EXPANSIONS / SELFCHECK_PROBABILITY / 2;
+    static_assert(min_correct_expansions > 0, "Increase minimal nof expansions");
+    static int64 num_expansions{min_correct_expansions};
+    num_expansions++;
+
     // We accept an error up to 5%, but output values
     // smaller than 1/1000th are "rounded up" for the comparison.
     constexpr float relative_error = 5e-2f;
     for (auto idx = size_t{0}; idx < data.size(); ++idx) {
         auto err = relative_difference(data[idx], ref[idx]);
-        if (err > relative_error) {
-            printf("Error in OpenCL calculation: expected %f got %f "
+        myprintf("Error in OpenCL calculation: expected %f got %f "
```

Comment:
I think this is going to print even if there is no error?

Reply:
The idea was to gather statistics on the error size.